### DISCUSSION TITLE
How to integrate ta.plot_light_curves(method='posterior') into a separate chart.

### DISCUSSION THREAD

**Discussion by jabienias** [hpparvi/PyTransit#156] [Q&A]
I have used Pytransit to model a transit using the phase bin flux midpoints. I would like to integrate the model into a chart that includes both the phase bin midpoints and the individual observations. How can I do that? I tried running evaluate (rrm('quadratic') with the posterior derived parameters, but it does not produce the same result (k needs to be larger to fit).  All suggestions welcome!

**Comment by jabienias** [comment]
I worked it out eventually.

**Reply by hpparvi** [reply]
Sorry, I forgot about this... Great to hear you got it working!
